gaming-website/src/javaScript/games.json

```diff
@@ -0,0 +1,184 @@
+        "price": "40",
+        "url": "../../public/assets/home-page/game-cover/the-witcher-3.png"
+      },
+      {
+        "name": "Hades",
+        "price": "25",
+        "url": "../../public/assets/home-page/game-cover/hades.png"
+      },
+      {
+        "name": "DOOM Eternal",
+        "price": "60",
+        "url": "../../public/assets/home-page/game-cover/doom.png"
+      },
+      {
+        "name": "Sekiro: Shadows Die Twice",
+        "price": "60",
+        "url": "../../public/assets/home-page/game-cover/sekiro.png"
+      },
+      {
+        "name": "Monster Hunter: World",
+        "price": "30",
+        "url": "../../public/assets/home-page/game-cover/monster-hunter.png"
+      },
+      {
+        "name": "Final Fantasy XIV: Shadowbringers",
+        "price": "40",
+        "url": "../../public/assets/home-page/game-cover/final-fantasy-XIV.png"
+      },
+      {
+        "name": "Control",
+        "price": "40",
+        "url": "../../public/assets/home-page/game-cover/control.png"
+      },
+      {
+        "name": "Death Stranding",
+        "price": "60",
+        "url": "../../public/assets/home-page/game-cover/death-stranding.png"
+      },
+      {
+        "name": "Star Wars Jedi: Fallen Order",
+        "price": "60",
+        "url": "../../public/assets/home-page/game-cover/star-wars-jedi.png"
+      },
+      {
+        "name": "Minecraft",
+        "price": "30",
+        "url": "../../public/assets/home-page/game-cover/minecraft.png"
+      },
+      {
+        "name": "Assassin's Creed Valhalla",
+        "price": "60",
+        "url": "../../public/assets/home-page/game-cover/assasins-creed-valhalla.png"
+      },
+      {
+        "name": "Borderlands 3",
+        "price": "60",
+        "url": "../../public/assets/home-page/game-cover/borderlands-3.png"
+      },
+      {
+        "name": "Ghost of Tsushima",
+        "price": "60",
+        "url": "../../public/assets/home-page/game-cover/ghosts-of-tsushima.png"
+      },
+      {
+        "name": "Nioh 2",
+        "price": "60",
+        "url": "../../public/assets/home-page/game-cover/Nioh-2.png"
+      },
+      {
+        "name": "Marvel's Spider-Man: Miles Morales",
+        "price": "50",
+        "url": "../../public/assets/home-page/game-cover/spider-man.png"
+      },
+      {
+        "name": "Resident Evil Village",
+        "price": "60",
+        "url": "../../public/assets/home-page/game-cover/resident-evil-village.png"
+      },
+      {
+        "name": "Demon's Souls",
+        "price": "70",
+        "url": "../../public/assets/home-page/game-cover/deamons-souls.png"
+      },
+      {
+        "name": "The Last of Us Part II",
+        "price": "60",
+        "url": "../../public/assets/home-page/game-cover/the-last-of-us.png"
+      }
+    ],
+
+    "gamesHighlight": [
+      {
+        "name": "Assassin's Creed Valhalla",
+        "url": "../../public/assets/home-page/highlights/assassins-creed-highlight.png"
```

Comment:
The format of the url și a bit weird

Reply:
I need to go up to levels from the json folder in orders for me to access the assets folder that's why i used `../../`, or how else could i go up 2 levels? or do you mean not go up 2 levels at all and restructure a second the structure file maybe?...

Reply:
or simply use JS to generate the url and have only the img name in here?

Reply:
It is probably weird because there seems to not be a server like apache or nginx. If it were, the root was already set to the public folder and you went from there straight to the images

Reply:
oh i see... 
Well we can simply generate the path from a JS file right?